android/src/com/frostwire/android/gui/services/EngineService.java

```diff
@@ -221,7 +221,7 @@ public static void updatePermanentStatusNotification(WeakReference<Context> cont
 
         // VPN status
         remoteViews.setImageViewResource(R.id.view_permanent_status_vpn_icon, isVPNactive ?
-                R.drawable.notification_vpn_on : R.drawable.notification_vpn_off);
+                R.drawable.notification_shutdown : R.drawable.notification_shutdown);
```

Comment:
so Gubatron wants to change the VPN icon to shutdown icon - changing the functionality of it too. Now you will be able to shutdown the app from the drop down notification. For now - just to preview the new icon I added it there. The functionality will need to be changed by him. So just a preview for now.